loop/storage/redis.js

```diff
@@ -3,7 +3,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 "use strict";
-var redis = require("redis");
+var redis = require("./redis_client");
```

Comment:
We a normal server we don't want to shutdown the multi feature. Was it a pain? really?